decidim-core/lib/decidim/core/engine.rb

```diff
@@ -559,6 +559,20 @@ class Engine < ::Rails::Engine
       config.to_prepare do
         FoundationRailsHelper::FlashHelper.include Decidim::FlashHelperExtensions
       end
+
+      config.after_initialize do
```

Comment:
To be consistent with the rest of initializers in this file, can you change this to a named initializer? Like
```suggestion
      initializer "decidim_core.preview_mailer" do
```

Reply:
> I tried this locally and it didn't work, I only see the "Decidim/Devise Mailer" previews:
> 
> ![image](https://user-images.githubusercontent.com/717367/156132500-8f02c768-5175-4d58-a27c-137cc5120a89.png)
> 
> Adding the same initiatizer to `decidim-initiatives/lib/decidim/initiatives/engine.rb` works, can you do that please? It should have another named initializer (like "deciidim_initiatives.preview_mailer")

huh! not sure it worked ytdy, but I wasn't able to reproduce it  anymore😆 . The idea was to let this run at the very end of the initializers process to not have to define it in other engines. Sadly it doesn't work consistently

Added it to the `decidim-initiatives` engine and it's working again 🙌🏽